### PR TITLE
Deprecate repo/package

### PR DESCRIPTION
**Pull request by pmeinhardt**
- Add deprecation message to README
- Add postinstall message

### CODE REVIEW

**Comment by pmeinhardt** [comment]
I suggest we just ditch the `postinstall` message?

**Comment by klappradla** [comment]
> I suggest we just ditch the `postinstall` message?

Yep. Not worth the effort if the "package" isn't used by anyone anyways 🤷

File: package.json [hunk 1]
```diff
@@ -13,7 +13,8 @@
     "lint:md": "prettier --ignore-path .gitignore --check '**/*.md'",
     "lint": "run-s --continue-on-error lint:js lint:json lint:md",
     "test": "jest",
-    "checks": "run-s lint test"
+    "checks": "run-s lint test",
+    "postinstall": "./script/postinstall"
```
Comment (klappradla) [inline package.json:17]:
```suggestion
    "checks": "run-s lint test"
```

As you said: perfectly fine to ditch this.

File: script/postinstall [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```
Comment (klappradla) [inline script/postinstall:1]: This file would then also need to go / could go 🗑️

**Review by klappradla — APPROVED** [review]
Ready to hit 🌊 without the post-install message 🏇

**Pull request merged**